lib/ddlog_std.rs

```diff
@@ -47,11 +47,12 @@ use std::{
     sync::Arc as StdArc,
     vec::{self, Vec as StdVec},
 };
-use twox_hash::{XxHash32, XxHash64};
+use xxhash_rust::{xxh3::Xxh3, xxh32::Xxh32};
 
 const XX_SEED1: u64 = 0x23b691a751d0e108;
 const XX_SEED2: u64 = 0x20b09801dce5ff84;
 
+#[inline]
```

Comment:
why don't we trust the compiler with all this inlining?

Reply:
We do, however this makes the functions viable for cross-crate inlining